runtime/src/governance/proposals.rs

```diff
@@ -320,6 +323,20 @@ decl_module! {
                 print(e);
             }
         }
+
+        /// Cancel a proposal and return stake without slashing
+        fn veto(proposal_id: u32) -> Result {
```

Comment:
Maybe rename to `veto_proposal`? I have next methods names:
- create_proposal
- vote_on_proposal
- cancel_proposal

So either rename these methods to simple form like `create`, `vote`, `cancel` or use longer but more descriptive style? I prefer more descriptive, thus you need fewer comments.